src/components/RequestsTable/FilterRequests.tsx

```diff
@@ -0,0 +1,334 @@
+          </Field>
+          <hr />
+          <Field label="Pay System">
+            <Combobox
+              name="paySystem"
+              clearable
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "paySystem";
+                })[0]
+                ?.filter.toString()}
+            >
+              {PAYSYSTEMS.map((system) => (
+                <Option key={system.key} value={system.key}>
+                  {system.key}
+                </Option>
+              ))}
+            </Combobox>
+          </Field>
+          <hr />
+          <Field label="Series">
+            <Input
+              name="series"
+              type="search"
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "series";
+                })[0]
+                ?.filter.toString()}
+              maxLength={4}
+            />
+          </Field>
+          <hr />
+          <Field label="Grade">
+            <Combobox
+              name="grade"
+              clearable
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "grade";
+                })[0]
+                ?.filter.toString()}
+            >
+              {ACQGRADES.map((grade) => (
+                <Option key={grade} value={grade}>
+                  {grade}
+                </Option>
+              ))}
+              {GENERALGRADES.map((grade) => (
+                <Option key={grade} value={grade}>
+                  {grade}
+                </Option>
+              ))}
+            </Combobox>
+          </Field>
+          <hr />
+          <Field label="Office Symbol">
+            <Input
+              name="officeSymbol"
+              type="search"
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "officeSymbol";
+                })[0]
+                ?.filter.toString()}
+            />
+          </Field>
+          <hr />
+          <Field label="Created By">
+            <Input
+              name="Author"
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "Author";
+                })[0]
+                ?.filter.toString()}
+            />
+          </Field>
+          <hr />
+          <Field label="Stage">
+            <Combobox
+              name="stage"
+              clearable
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "stage";
+                })[0]
+                ?.filter.toString()}
+            >
+              {STAGES.map((stage) => (
+                <Option key={stage.key} text={stage.key} value={stage.key}>
+                  {stage.text}
+                </Option>
+              ))}
+            </Combobox>
+          </Field>
+          <hr />
```

Comment:
I assume you are still looking into how to do it, but need a way to clear the Date field.  I sadly admit it took me a second to see the "X" on the Combobox's to clear them, as I was looking for a "blank" entry to select.

Reply:
The Clear button for Date fields was not working for me.  The Clear All wasn't clearing them either.  Other than that, I like the addition.